content/components/skip-link/default.md

```diff
@@ -21,8 +29,18 @@ code:
           text: 'Skip to main navigation',
         },
       ]} />
+      <nav id="nav" tabindex="-1">
+        <ul>
+          <li><a href="#">Some navigation</a></li>
+        </ul>
+      </nav>
+      <div id="content" tabindex="-1">
+        Some content here
+      </div>
 ---
 
 Skip links are not visual until they receive focus.
 
 Include skip links between the opening of the `<body>` and the page `<header>`.
+
+The content that the skip links goes to is linked with and `id` and it moves the focus to the element with `tabindex="-1"`.
```

Comment:
Little typo to fix up:

> The content that the skip links goes to is linked with **an** `id` and it moves the focus to the element with `tabindex="-1"`.

Reply:
@alex-page does this description go far enough to resolve https://github.com/govau/uikit/issues/389 ?

Should there be an explicit recommendation to add `tabindex="-1"` to the destination anchor?

Reply:
I think it does, I updated the examples to copy and paste aswell